### ISSUE TITLE
CLI error after upgrading to v0.10.0

### ISSUE THREAD

**Issue by Perdolique** [nuxt/fonts#328]
I've upgraded `@nuxt/fonts` to `v0.10.0` and after running `npm run dev` receiving the following errors:

```
[10:08:40 PM]  ERROR  Could not resolve font face for Segoe UI from adobe provider. Cannot read properties of undefined (reading 'resolveFont')
[10:08:40 PM]  ERROR  Could not resolve font face for SFMono-Regular from adobe provider. Cannot read properties of undefined (reading 'resolveFont')
[10:08:40 PM]  ERROR  Could not resolve font face for Inter from adobe provider. Cannot read properties of undefined (reading 'resolveFont')
```

I've tried to set `adobe: false` in the config, but still getting these errors. I'm using `Inter` font only: `font-family: "Inter", sans-serif;`

![Image](https://github.com/user-attachments/assets/ab571ce6-3f0a-4b2d-bb5d-5293fbbc5df1)

**Comment by Mateleo** [comment]
Same error for me with v0.10.0
![Image](https://github.com/user-attachments/assets/e510aced-be58-4b32-a699-9744bc8efda6)

**Comment by IzakJackson** [comment]
Same for me

**Comment by danielroe** [comment]
Apologies for this! Can you create a reproduction? 🙏

**Comment by Perdolique** [comment]
@danielroe sure, it's [just a default use](https://stackblitz.com/edit/nuxt-starter-mlwm87?file=app.vue) w/o any specific configuration.

**Comment by danielroe** [comment]
this should be resolved in v0.1.3 of unifont: https://github.com/unjs/unifont/releases/tag/v0.1.3, which you can get by refreshing your lockfile or running `npm upgrade unifont`

my apologies!

thank you for reporting...
